src/test/kotlin/uk/gov/justice/digital/hmpps/prisonregister/service/PrisonServiceTest.kt

```diff
@@ -342,4 +345,45 @@ class PrisonServiceTest {
         .isInstanceOf(ContactDetailsNotFoundException::class.java)
     }
   }
+
+  @Nested
+  inner class CreateContactDetails {
+
+    private val prisonRepository2: PrisonRepository = mock()
```

Comment:
do you really need to create a new one?

Reply:
I have added a commit to reset repository mocks https://github.com/beforeeach this fix the issue of reusing them. this apply to all 3 PR opened with similar tests.  I have **cherry-pick** same commit in all 3 PR.